### DISCUSSION TITLE
how to tell traefik to deafult redirect 301 instead of 302 in coolify v3 ??

### DISCUSSION THREAD

**Discussion by nianik5098** [coollabsio/coolify#3149] [Q&A]
happy coolify v3 user here. 
coolify v3 by default redirects http -> https as 302 (temporary redirect) which is not good for SEO.
I want to configure the traefik container which comes bundled with coolify v3 to use 301 redirects by default.
where do i start? 
please help.

**Comment by peaklabs-dev** [comment]
Can you upgrade to v4 and try again (since v3 no longer receives updates)?

**Reply by nianik5098** [reply]
does that mean i cannot tweak the traefik which comes bundled with coolify v3 ??
i am not afraid of command lines but don't have enough experiences with docker containers. 
if someone can show me a direction, i can dive deeper.
(thanks for replying).

**Comment by RizqiSyahrendra** [comment]
@nianik5098 so did you find a way to do this?

**Reply by nianik5098** [reply]
Hello there,
I don't remember exactly what i did.
Look for traefik's syntax how it configures 301 or 302 redirects.
Then search in the coolify's codebase for the file that contains the http redirects.
Try to find that file or a similarly named file in your server's file system. Or do string search on your server's file system using the 302 redirect syntax.
If you find the desired file, edit the code block that has the redirection definition for your desired container. (restart if needed).

This low quality hack might break during coolify version upgrades.

**Comment by djsisson** [comment]
see here:

https://github.com/coollabsio/coolify/issues/5126#issuecomment-2654270887
